README.md

```diff
@@ -34,20 +48,64 @@ You can run the local UI through [joshsim-server](https://language.joshsim.org/d
 $ java -jar joshsim-server.jar
 ```
 
-This will start a local web server which makes the UI available via your browser where you can work in private.
+This will start a local web server which makes the UI available via your browser where you can work in private. The local server optionally supports a sandbox mode that limits access to only the code and jshd files provided, with no network access otherwise.
 
 ### Containerized usage
 Containerization through [Docker](https://www.docker.com) and [Development Containers](https://containers.dev) can help you move your work from one computer to the next with ease. Please see our `Dockerfile` and `devcontainer.json`.
 
 ### Distributed usage
-Distributing workloads is easy. Simply deploy either our jar file or container to a serverless solution like [Labmda](https://aws.amazon.com/lambda/) / [CloudRun](https://cloud.google.com/run) or submit on your own cluster via [Kubernetes](https://kubernetes.io). You can send the Josh jar over the network to get your script and you can write to cloud storage. All you have to do is provide the command line arguments:
+Distributing workloads is easy. Simply deploy either our jar file or container to a serverless solution like [Lambda](https://aws.amazon.com/lambda/) / [CloudRun](https://cloud.google.com/run) or submit on your own cluster via [Kubernetes](https://kubernetes.io). The Josh server supports HTTP2 for efficient communication. You can send the Josh jar over the network to get your script and you can write to cloud storage. All you have to do is provide the command line arguments:
```

Comment:
Please expand this to two examples.

 - The first should not specify an output and please indicate that it returns the results through HTTP2 streaming back to the user client.
 - The second example should include the output location via `--output` but please indicate that, at this time, avro dumping is limited.